aws/alb-ingress-iam-policy.json

```diff
@@ -0,0 +1,140 @@
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Action": [
+        "acm:DescribeCertificate",
+        "acm:ListCertificates",
+        "acm:GetCertificate"
+      ],
+      "Resource": "*"
+    },
+    {
+      "Effect": "Allow",
+      "Action": [
+        "ec2:AuthorizeSecurityGroupIngress",
```

Comment:
Are these actions shared by Fargate? Or does the ALB have to run on EC2 under the hood?

Reply:
There's no EC2, the alb runs on fargate:

```
timothe-macbookpro~/healthmap/healthmap-gdo-temp(alb|✔) % kubectl get nodes
NAME                                      STATUS   ROLES    AGE     VERSION
fargate-ip-192-168-108-133.ec2.internal   Ready    <none>   21h     v1.15.10-eks-094994
fargate-ip-192-168-110-175.ec2.internal   Ready    <none>   21h     v1.15.10-eks-094994
fargate-ip-192-168-117-236.ec2.internal   Ready    <none>   5d19h   v1.15.10-eks-094994
fargate-ip-192-168-124-31.ec2.internal    Ready    <none>   5d19h   v1.15.10-eks-094994
fargate-ip-192-168-83-164.ec2.internal    Ready    <none>   21h     v1.15.10-eks-094994
fargate-ip-192-168-93-139.ec2.internal    Ready    <none>   21h     v1.15.10-eks-094994
fargate-ip-192-168-95-118.ec2.internal    Ready    <none>   4d23h   v1.15.10-eks-094994
timothe-macbookpro~/healthmap/healthmap-gdo-temp(alb|✔) % kubectl get pods -n kube-system
NAME                                      READY   STATUS    RESTARTS   AGE
alb-ingress-controller-7ff58dffbf-zf8qs   1/1     Running   0          21h
coredns-55bdcf5695-4gtlz                  1/1     Running   0          5d19h
coredns-55bdcf5695-jbdhh                  1/1     Running   0          5d19h
timothe-macbookpro~/healthmap/healthmap-gdo-temp(alb|✔) % kubectl describe pod alb-ingress-controller-7ff58dffbf-zf8qs -n kube-system
Name:                 alb-ingress-controller-7ff58dffbf-zf8qs
Namespace:            kube-system
Priority:             2000001000
Priority Class Name:  system-node-critical
Node:                 fargate-ip-192-168-93-139.ec2.internal/192.168.93.139
Start Time:           Tue, 02 Jun 2020 12:51:21 +0200
Labels:               app.kubernetes.io/name=alb-ingress-controller
                      eks.amazonaws.com/fargate-profile=fp-default
                      pod-template-hash=7ff58dffbf
```